R/mod_chartsTab.R

```diff
@@ -27,7 +27,7 @@ chartsTabUI <- function(id, chart){
 #' 
 #' @export
 
-chartsTab <- function(input, output, session, chart, data, mapping){  
+chartsTab <- function(input, output, session, chart, data, mapping, module_outputs) {
```

Comment:
Is the idea that this is all of outputs from the other charts? Is this reactive? Are you using this for anything right now? Wondering if we could trigger an infinite loop with this somehow ...

Reply:
I'm imagining a module that aggregates output from other modules, each with its own set of inputs.